### DISCUSSION TITLE
Indicate when tailwind class does not exist

### DISCUSSION THREAD

**Discussion by pbrzosko** [tailwindlabs/tailwindcss#12912] [Help]
Hi,

I have created a bug: https://github.com/tailwindlabs/tailwindcss/issues/12908.

I would love to have a way to know, that I use a class in Vue template that doesn't exist. I see the answer to use a prettier plugin, so that classes get sorted and then I can check the ones at the beginning. This is a no go for us though, as the project is to big to check all the classes.
We defined tailwind config from our design tokens and they change a lot. We need to have a way to figure out, that names of design tokens has changed, and that then we need to update our codebase. Otherwise this is a lot of searching, greping and renaming.

Is there a way we could maybe write a plugin to eslint / tailwind / postcss so that we will get notified about using of a class, that is not defined. in tailwind?

**Comment by epavanello** [comment]
Same need here
